mypy_django_plugin/transformers/managers.py

```diff
@@ -67,6 +67,11 @@ def create_new_manager_class_from_from_queryset_method(ctx: DynamicClassDefConte
             break
         for name, sym in class_mro_info.names.items():
             if isinstance(sym.node, FuncDef):
-                helpers.copy_method_to_another_class(
-                    class_def_context, self_type, new_method_name=name, method_node=sym.node
-                )
+                func_node = sym.node
+            elif isinstance(sym.node, Decorator):
```

Comment:
We totally need a test for this!

Reply:
Good call, I'll work on adding one tonight 👍

Reply:
The test I added fails in this way on master:
```
________________________ from_queryset_with_decorated_queryset_methods ________________________
/Users/patrick/code/django-stubs/tests/typecheck/managers/querysets/test_from_queryset.yml:187: 
E   pytest_mypy_plugins.utils.TypecheckAssertionError: Invalid output: 
E   Expected:
E     main:2: note: Revealed type is "myapp.models.MyModel_NewManager[myapp.models.MyModel]"
E     main:3: note: Revealed type is "builtins.str" (diff)
E   Actual:
E     main:2: note: Revealed type is "myapp.models.MyModel_NewManager[myapp.models.MyModel]"
E     main:3: error: "MyModel_NewManager[MyModel]" has no attribute "queryset_method" (diff)
E     main:3: note: Revealed type is "Any"          (diff)
E   
E   Alignment of first line difference:
E     E: main:3: note: Revealed type is "builtins.str"...
E     A: main:3: error: "MyModel_NewManager[MyModel]" has no attribute "queryset_...
E                ^
```